client/src/cl_cvarlist.cpp

```diff
@@ -455,8 +455,8 @@ CVAR_RANGE(		hud_fullhudtype, "1","Fullscreen HUD to display:\n// 0: ZDoom HUD\n
 CVAR_RANGE(		hud_gamemsgtype, "2", "Game message type",
 				CVARTYPE_BYTE, CVAR_CLIENTARCHIVE | CVAR_NOENABLEDISABLE, 1.0f, 2.0f)
 
-CVAR(			hud_revealsecrets, "1", "Print HUD message when secrets are discovered",
-				CVARTYPE_BOOL, CVAR_CLIENTARCHIVE)
+CVAR(			hud_revealsecrets, "1", "Print a HUD message and|or play a sound when a secret is discovered.",
+				CVARTYPE_BYTE, CVAR_CLIENTARCHIVE | CVAR_NOENABLEDISABLE, 0.0f, 3.0f)
```

Comment:
This is failing the build on Linux - I think this needs to be a `CVAR_RANGE`.

Reply:
Oops, that was something I failed to see! Thanks for noticing it!